input/intro-notes/StructureDefinition-au-practitioner-intro.md

```diff
@@ -42,7 +42,7 @@
     <td>"document"</td>
     <td>SHALL use an appropriate value from <a href="ValueSet-rsg-source-document-type.html">AU Recorded Sex or Gender (RSG) Source Document Type</a> value set if any of the codes within the value set can apply to the concept being communicated, e.g. "passport".</td>
     <td>The name of the field within the source document where this information is recorded, e.g. "Sex".</td>
-    <td>SHALL use an appropriate value from <a href="ValueSet-rsg-source-document-jurisdiction.html">AU Recorded Sex or Gender (RSG) Source Document Jurisdiction</a> value set if any of the codes within the value set can apply to the concept being communicated. If representing an Australian document use "AU" or the applicable state or territory code.</td>
+    <td>SHALL use an appropriate value from <a href="valueset-au-jurisdiction-extended.html">Jurisdiction - AU Extended</a> value set if any of the codes within the value set can apply to the concept being communicated. If representing an Australian document use "AU" or the applicable state or territory code.</td>
```

Comment:
IG publisher isn't recognising the url with that case. Replace with "ValueSet-au-jurisdiction-extended.html"

Reply:
Updated valueset page name to Title case